frontend/src/scenes/notebooks/Nodes/NotebookNodeFlagLogic.tsx

```diff
@@ -39,5 +43,18 @@ export const notebookNodeFlagLogic = kea<notebookNodeFlagLogicType>([
                     )
                 },
         ],
+        shouldDisableInsertSurvey: [
+            (s) => [s.featureFlag, s.hasSurveys],
+            (featureFlag, hasSurveys) =>
+                (nextNode: Node | null): boolean => {
+                    return (
+                        (nextNode?.type.name === NotebookNodeType.Survey &&
+                            hasSurveys &&
+                            featureFlag.surveys &&
```

Comment:
Do we need this check given we're already guaranteeing surveys exist as part of the `hasSurveys` selector

Reply:
just satisfying typing 🙈 , because it can't parse that hasSurveys implies surveys exist